src/terms/terms.jl

```diff
@@ -99,9 +99,13 @@ function compute_kernel(basis::PlaneWaveBasis{T}; kwargs...) where {T}
     end
     k
 end
-function apply_kernel(basis, dρ; kwargs...)
+
+function apply_kernel(basis, dρ; rpa=false, kwargs...)
```

Comment:
Caps for RPA?

Reply:
Hmmm we don't really have all-caps kwargs so far and I'm not so much a fan of it, actually.

Reply:
We usually use caps for abbrevations (eg DFT, LDA - except in libxc), so we might just as well do it here also no?